### PR TITLE
Updated launcher tool installation process. Fixes RHAORDOC-299

### PR DESCRIPTION
**Pull request by rhoads-zach**
Blocked until this is merged: 
https://github.com/minishift/minishift-addons/pull/124

### CODE REVIEW

File: docs/topics/proc_installing-launcher-tool-addon.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+$ minishift addons apply fabric8-launcher \ 
+      --addon-env GITHUB_USERNAME=GH_USERNAME --addon-env GITHUB_TOKEN=TOKEN <1>
+----
+<1> Use your GitHub username and personal access token
+
+. Monitor the status of the {launcher} tool until it completes start up.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc project rhoarpad
```
Comment (Ladicek) [inline docs/topics/proc_installing-launcher-tool-addon.adoc]: Might want to explain that the addon creates a new project, defaulting to the name `rhoarpad`.

(Which, frankly, is silly, but whatevz...)

**Comment by rhoads-zach** [comment]
@Ladicek updated

**Comment by inoxx03** [comment]
LGTM, merging...

**Comment by inoxx03** [comment]
merged: d1f89bbeee783895a5cb7b0b916fc371b816865e

File: docs/topics/proc_installing-launcher-tool-addon.adoc (hunk 1 above)
Comment (inoxx03) [reply docs/topics/proc_installing-launcher-tool-addon.adoc]: @lthon I think that information is evident form the note on line #29. WDYT?

Comment (Ladicek) [reply docs/topics/proc_installing-launcher-tool-addon.adoc]: It is now. It wasn't when I added the comment :-)

(In other words: this comment has been addressed.)

Comment (inoxx03) [reply docs/topics/proc_installing-launcher-tool-addon.adoc]: Nice:)  thank you:)

**Pull request merged**